README.md

```diff
@@ -85,12 +117,415 @@ return (
+
+```typescript
+const {controller} = createPromoManager({
+    config: {
+        promoGroups: [
+            {
+                slug: '1',
+                promos: [
+                    {
+                        slug: 'promo1',
+                        conditions: [MatchUrl('/folder/\\w{5}/page$'),],
+                        trigger: {on: 'pageOpened', timeout: 2000},
+                    },
+                ],
+            },
+        ],
+    },
+    plugins: [new UrlEventsPlugin({eventName: 'pageOpened'})],
+})
+```
+
+## JSON config
+
+You can define conditions, constraints as JSON serializable objects. So you can take congig from json, parse it and use. It can be useful for editing config without rebuild project and release.
+
+
+```typescript
+
+const usersWithLanguage = (language) => user.language = language;
+
+const {controller} = createPromoManager({
+    config: { // config section now can be parsed from json
+        constraints: [
+            {
+                helper: 'LimitFrequency',
+                args: [{
+                    slugs: ['somePoll', 'groupOfHints'], // can use promos slugs and group slugs
+                    interval: {days: 1},
+                }],
+            },
+        ],
+        promoGroups: [{
+            slug: 'groupOfPolls',
+            promos: [
+                {
+                    slug: 'somePoll',
+                    conditions: [{
+                        helper: 'ShowOnceForPeriod',
+                        args: [{month: 1}]
+                    }]
+                },
+                {
+                    slug: 'pollForPageWithparam',
+                    conditions: [
+                        {
+                            helper: 'MatchUrl',
+                            args: ['param=value']
+                        },
+                        {
+                            helper: 'usersWithLanguage',
+                            args: ['English']
+                        },
+                    ]
+                }
+            ],
+        }]
+    },
+    conditionHelpers: {
+        usersWithLanguage,
+    },
+});
 ```
 
-## Usage
+## Onboarding integration
+
+You can use onboarding with `PromoPresetsPlugin` to show advertising and educational hints. You can use promo manager to limit frequency and set constraint with other promo in service.
 
-TBD
+```typescript
+import {
+    createOnboarding
+} from "./index";
+
+const {controller: onboardingController} = createOnboarding({
+    config: {
+        presets: {
+            coolNewFeature: {
+                name: 'Cool feature',
+                visibility: 'alwaysHidden',
+                steps: [/**/]
+            },
+            coolNewFeature2: {
+                name: 'Cool feature2',
+                visibility: 'alwaysHidden',
+                steps: [/**/],
+            }
+        }
+    },
+    plugins: [new PromoPresetsPlugin(),]
+   /**/ 
+})
+
+const {controller} = createPromoManager({
```

Comment:
is it possible to use `promoManager` with regular onboarding presets (not `alwaysHidden`)?

Reply:
It is not possible. Because regular onboarding presets is not promo activity. We dont push it into user. The user launches it himself.
But it can can be changed later. For example to prevent start promos during onboarding